README.md

```diff
@@ -35,16 +47,22 @@ getGeoJSON(osm, function (err, geojson) {
 var getGeoJSON = require('osm-p2p-geojson')
 ```
 
-### var rstream = getGeoJSON(osm[, options][, callback])
+### var stream = getGeoJSON(osm[, options][, callback])
 
 Get GeoJSON from the database. GeoJSON is returned as a readable stream, or, if passed, `callback(err, geoJson)`.
```

Comment:
Creates a TransformStream that will take as input a stream of osm-p2p documents and outputs a stream of GeoJSON. If you prefer a callback rather than a stream as output you can pass `callback(err, geoJson)`.